db/seeds.rb

```diff
@@ -39,3 +39,12 @@
     end
   end
 end
+<<<<<<< HEAD
```

Comment:
Weird stuff you have...

Nepareizi samergoji. Pielabo lai būtu ta pat kā master branča.